packages/components/src/components/Checkbox/Checkbox.tsx

```diff
@@ -0,0 +1,263 @@
+import { Pressable, StyleProp, Text, View, ViewStyle } from 'react-native'
+import { spacing } from '@department-of-veterans-affairs/mobile-tokens'
+import { useTranslation } from 'react-i18next'
+import React, { FC } from 'react'
+
+import { ComponentWrapper } from '../../wrapper'
+import { Icon } from '../Icon/Icon'
+import { Spacer } from '../Spacer/Spacer'
+import { useTheme } from '../../utils'
+
+export type CheckboxProps = {
+  /** Primary text for checkbox */
+  label: string
+  /** OnPress function. Pass a function that alters `checked` state */
+  onPress: () => void
+  /** Optional boolean that determines whether checkbox appears checked */
+  checked?: boolean
+  /** Optional description that appears below label */
+  description?: string
+  /** Optional accessibility override for description text  */
+  descriptionA11y?: string
```

Comment:
With how much a11y label overriding we have going here, I wonder if a better solution might be something like:
```
type textWithA11y = {
  text: string
  a11y: string
}
```

and typing this like:
```suggestion
  description?: string | textWithA11y
```

Would add some complexity using the values though, might be worth playing with locally and seeing if it shakes out to be beneficial to simplify the props from being so cluttered with text + a11y combos.

Reply:
> I think this component probably has some of the most individual text props we'll encounter and it may be okay as is.

Seems like Radio will probably be the same and other form components will have at least 2 (label, error) if not more.

Agree with the general notion it should be applied to existing components, but could also see a caveat of `only applies to components with more than one "text" value` which would mean just this and Alert (and technically Segmented control, but that's using arrays for the labels and corresponding a11y so it's still "1" in a sense).

I wonder if we should start making a list of items to target/consider as part of going version 1.0.0 where we can feel free to make widespread breaking changes when it's expected as opposed to the current situation where flagship is largely hopping component-to-component version with no regard to what else we're up to.

All that said, this would be mostly for our benefit and not do much for consumers so maybe it's fine, if a bit obnoxious to look at.